code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -880,6 +880,7 @@
 /datum/reagent/medicine/strange_reagent/instant
 	name = "Stranger Reagent"
 	instant = TRUE
+	chemical_flags = NONE
```

Comment:
```suggestion
	chemical_flags = REAGENT_NO_RANDOM_RECIPE
```

Reply:
still needs the flag for not being allowed in random recipes

Reply:
![image](https://github.com/tgstation/tgstation/assets/6209658/360bfbe4-653a-49b6-a321-3db60c006efc)
no you don't.